<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alterar_cadastro</class>
 <widget class="QDialog" name="alterar_cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALTERAR CADASTRO LEITOR</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NOME </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NOME">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TELEFONE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="TELEFONE">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CPF </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CPF">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PESQUISAR</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>INDICE LEITOR </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="INDICE">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ALTERAÇÃO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="VALOR_ALTERADO">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CAMPO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CAMPO">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Nome_leitor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Telefone</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rua</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Numero</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bairro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cidade</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Estado</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CEP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>E_mail</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CPF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RG</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ALTERAR CAMPO</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
